src/hooks/errors.js

```diff
@@ -1,12 +1,13 @@
 import { SINGLE_ERROR_CLEAR } from 'actions/actionTypes';
 import { useDispatch, useSelector } from 'react-redux';
+import { selectErrorByAction } from 'selectors/clusterSelectors';
 import { typeWithoutSuffix } from 'selectors/selectorUtils';
 
 export const useError = (errorType) => {
   const dispatch = useDispatch();
 
   return useSelector((state) => {
```

Comment:
Just wondering, why do you return the output of `useSelector` like that?

Why not 
```javascript
  const dispatch = useDispatch();
  const errorMessage = useSelector(state =>
    selectErrorByAction(state, errorType)
  );

  const clear = () => {
    dispatch({
      type: SINGLE_ERROR_CLEAR,
      errorType: typeWithoutSuffix(errorType)
    });
  };

  return { errorMessage, clear };
};
```

Does it provide any additional benefit?

Reply:
Even I'm not sure what I was smoking

Reply:
Cleaned it up :)